test/load_stnu.py

```diff
@@ -0,0 +1,114 @@
+import unittest
+import json
+import logging
+import sys
+from stn.stp import STP
+import os
+
+code_dir = os.path.abspath(os.path.dirname(__file__))
+STNU = code_dir + "/data/stnu_two_tasks.json"
+MAX_FLOAT = sys.float_info.max
+
+logger = logging.getLogger()
+logger.level = logging.DEBUG
+stream_handler = logging.StreamHandler(sys.stdout)
+logger.addHandler(stream_handler)
+
+
+class LoadSTNU(unittest.TestCase):
```

Comment:
Is the test really testing how to load the STNU? It seems to me that the test is doing so much more